docs/deprecations.md

```diff
@@ -19,11 +19,21 @@ being deprecated.
 | [PipelineRun.Timeout is deprecated and will be removed](https://github.com/tektoncd/community/blob/main/teps/0046-finallytask-execution-post-timeout.md)                                                                     | v0.36.0                                                              | Beta                                                                                                   | Feb 25, 2023                        |
 | [Several fields of Task.Step are deprecated](https://github.com/tektoncd/pipeline/issues/4737)                                                                                                                               | v0.36.0                                                              | Beta                                                                                                   | Feb 25, 2023                        |
 | [`v1alpha1.Run` is deprecated](https://github.com/tektoncd/community/blob/main/teps/0114-custom-tasks-beta.md)                                                                                                               | v0.43.0                                                              | Alpha                                                                                                  | April 10, 2023 or v0.47.0           |
 | [ClusterTask is deprecated](https://github.com/tektoncd/pipeline/issues/4476)                                                                                                                                                | v0.41.0                                                              | Beta                                                                                                   | July 13, 2023                       |
 | [`pipelineRef.bundle` and `taskRef.bundle` are deprecated](https://github.com/tektoncd/pipeline/issues/5514)                                                                                                                 | v0.41.0                                                              | Alpha                                                                                                  | July 13, 2023                       |
-| [`taskrun.status.cloudEvents` are deprecated](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)                                                                                     | v0.44.0                                                              | Alpha                                                                                                  | Oct 11, 2023                        |
+
+### Deprecate PipelineResources related features:
```

Comment:
suggest "Deprecate**d** PipelineResources related features:" instead of "Deprecate PipelineResources related features:" to be clear

Reply:
Makes sense!